mapfishapp/src/main/webapp/app/addons/notes/js/main.js

```diff
@@ -0,0 +1,149 @@
+Ext.namespace("GEOR.Addons");
+
+GEOR.Addons.Notes = Ext.extend(GEOR.Addons.Base, {
+
+    toggleGroup: "notes",
+
+    /**
+     * Method: init
+     */
+    init: function(record) {
+        if (this.target) {
+            // addon placed in toolbar
+            this.components = this.target.insertButton(this.position, {
+                xtype: "button",
+                enableToggle: true,
+                toggleGroup: this.toggleGroup,
+                tooltip: this.getTooltip(record),
+                icon: GEOR.config.PATHNAME + "/ws/addons/" +
+                    record.get("name").toLowerCase() + "/" +
+                    this.options.icon,
+                listeners: {
+                    "toggle": this.onToggle,
+                    scope: this
+                },
+                scope: this
+            });
+            this.target.doLayout();
+        } else {
+            // addon places in "tools menus"
+            this.item = new Ext.menu.CheckItem({
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["Ext" is not defined.](https://www.codacy.com/app/georchestra/georchestra/file/2331320306/issues/source?bid=3200181&fileBranchId=3206983#l30)